test/e2e/tests/tolerate-failure/onboarding/onboarding-import-wallet.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { Mockttp } from 'mockttp';
+import { WALLET_PASSWORD, withFixtures } from '../../../helpers';
+import { Driver } from '../../../webdriver/driver';
+import HomePage from '../../../page-objects/pages/home/homepage';
+import OnboardingCompletePage from '../../../page-objects/pages/onboarding/onboarding-complete-page';
+import OnboardingMetricsPage from '../../../page-objects/pages/onboarding/onboarding-metrics-page';
+import OnboardingPasswordPage from '../../../page-objects/pages/onboarding/onboarding-password-page';
+
+import StartOnboardingPage from '../../../page-objects/pages/onboarding/start-onboarding-page';
+
+import OnboardingSrpPage from '../../../page-objects/pages/onboarding/onboarding-srp-page';
+import { ALL_POPULAR_NETWORKS } from '../../../../../app/scripts/fixtures/with-networks';
+import FixtureBuilder from '../../../fixture-builder';
+import HeaderNavbar from '../../../page-objects/pages/header-navbar';
+import AccountListPage from '../../../page-objects/pages/account-list-page';
+import { getCommonMocks } from '../utils/commonMocks.js';
+import { setupTimerReporting } from '../utils/testSetup.js';
+import Timers from '../../timers/Timers.js';
+
+const E2E_SRP = process.env.E2E_POWER_USER_SRP || 'srp test phrase'; // default SRP for testing
```

Comment:
Sorry, I meant `E2E_SRP` as in `import { E2E_SRP } from '../../../default-fixture.js';`

Then below it can be `process.env.E2E_POWER_USER_SRP || E2E_SRP`